eve/main.yml

```diff
@@ -2,18 +2,31 @@
 version: 0.2
```

Comment:
Can we use the latest version of eve?

Reply:
@rahulreddy checked with @tcarmet v0.2 is the latest